### ISSUE TITLE
Can't change language

### ISSUE THREAD

**Issue by kaduleite** [asakous/Neo-EmuELEC-H3#60]
I change language in settings, but it has no effect.

**Comment by asakous** [comment]
after language changed  . did you reboot the system ?

**Comment by kaduleite** [comment]
Yes, I reboot.  It's a strange behavior, because I installed Akekful theme and theme preferences are in Portuguese, but the system is still in English.

**Comment by asakous** [comment]
since emulationstation package is inherited from EmuELEC  and I currently use old version of emulationstation . 
maybe it's a EmuELEC's  emulationstation bug or lack of Portuguese translation. 
I will try to upgrade to newer version of emulationstation .

**Comment by kaduleite** [comment]
Thank you very much.

**Comment by eltonds** [comment]
Same problem here. Do you still plan to update this version and try to fix this issue?
